src/Tasks/Microsoft.Common.Test.targets

```diff
@@ -0,0 +1,28 @@
+<!--
+***********************************************************************************************
+Microsoft.Common.Test.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (C) Microsoft Corporation. All rights reserved.
+***********************************************************************************************
+-->
+<!--
+    ============================================================
+    This stub `Test` target allows for targets implementing Test execution functionality
+    to run after it.
+    
+    For example:
+    <Target Name="RunVSTest" AfterTargets="Test">
+     (implementation)
+    </Target>
+    ============================================================
+  -->
+<Project>
+    <PropertyGroup>
+        <UseMSBuildTestInfrastructure>false</UseMSBuildTestInfrastructure>
```

Comment:
I think that @rainersigwald said that's better have the condition in the import

Reply:
Good point, that would avoid the full target to be loaded.

Reply:
Updated